frontend-project/src/models/institutions.ts

```diff
@@ -0,0 +1,54 @@
+import { fetchAll } from '@/services/institutions';
+import { Effect, EffectsCommandMap } from 'dva';
+import { AnyAction } from 'redux';
+import { Reducer } from 'react';
```

Comment:
Reducer z Reduxa jest lepiej zrobiony jak dobrze kojarzę (ale w gruncie rzeczy to prawie ten sam interfejs)